lib/rspec/core/configuration.rb

```diff
@@ -870,7 +874,7 @@ def full_description
       # Adds a formatter to the set RSpec will use for this run.
       #
       # @see RSpec::Core::Formatters::Protocol
-      def add_formatter(formatter, output=output_stream)
+      def add_formatter(formatter, output=output_wrapper)
```

Comment:
It’s possible to send specific output stream to `add_formatter`:

output = StringIO.new
RSpec.configure do |c|
  c.add_formatter :documentation
  c.add_formatter :progress, output  
  c.output_stream = File.open('saved_output', 'w')
end

In this case `c.output_stream = File.open('saved_output', 'w')` should change output stream for `:documentation` formatter, but not for `:progress` formatter.

So i suppose,  it’s not obligatorily to wrap output param in add_formatter. Anyway maybe it worth to wrap output param here, in that case formatters always will operate with an object of `OutputWrapper`.